templates/zerver/help/index.md

```diff
@@ -126,6 +126,7 @@ as an **organization**.
 * [Allow anyone to join without an invitation](/help/allow-anyone-to-join-without-an-invitation)
 * [Only allow admins to invite new users](/help/only-allow-admins-to-invite-new-users)
 * [Only allow admins to create new streams](/help/only-allow-admins-to-create-new-streams-feature)
+* [Prevent users from changing their name](/help/disallow-users-to-change-their-name)
```

Comment:
The title of the article should match the URL.  I think the URL is worse so you switch that.

Also, add a mention of this feature on the "change your name" part?